### PR TITLE
boops: init at 1.2.0

### PR DESCRIPTION
**Pull request by magnetophon**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/audio/boops/default.nix [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  version = "1.2.0";
+
+  src = fetchFromGitHub {
+    owner = "sjaehn";
+    repo = pname;
+    rev = version;
+    sha256 = "0mhb0x1rdf1bcqnralg12mdpd7ihzv8ifzp9xv5c2ybcsjx4d9ad";
+  };
+
+  nativeBuildInputs = [ pkgconfig ];
```
Comment (SuperSandro2000) [inline pkgs/applications/audio/boops/default.nix:14]:
```suggestion
  nativeBuildInputs = [ pkg-config ];
```

File: pkgs/applications/audio/boops/default.nix [hunk 2]
```diff
@@ -0,0 +1,28 @@
+{ stdenv, fetchFromGitHub, xorg, cairo, lv2, libsndfile, pkgconfig }:
```
Comment (SuperSandro2000) [inline pkgs/applications/audio/boops/default.nix:1]:
```suggestion
{ stdenv, fetchFromGitHub, xorg, cairo, lv2, libsndfile, pkg-config }:
```

File: pkgs/applications/audio/boops/default.nix [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+  };
+
+  nativeBuildInputs = [ pkgconfig ];
+  buildInputs = [
+    xorg.libX11 cairo lv2 libsndfile
+  ];
+
+  installFlags = [ "PREFIX=$(out)" ];
+
+  meta = with stdenv.lib; {
```
Comment (SuperSandro2000) [inline pkgs/applications/audio/boops/default.nix:21]:
```suggestion
  meta = with lib; {
```

File: pkgs/applications/audio/boops/default.nix [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+    homepage = "https://github.com/sjaehn/BOops";
+    description = "Sound glitch effect sequencer LV2 plugin";
+    maintainers = [ maintainers.magnetophon ];
+    platforms = platforms.linux;
+    license = licenses.gpl3;
```
Comment (SuperSandro2000) [inline pkgs/applications/audio/boops/default.nix:26]: Only or Plus?

**Review by SuperSandro2000 — CHANGES_REQUESTED** [review]

**Comment by magnetophon** [comment]
@SuperSandro2000 @fabaff 
I accidentally closed this PR while resolving your issues.
New PR here: https://github.com/NixOS/nixpkgs/pull/123940

**Pull request closed without merging**